scheduler/src/cook/rest/api.clj

```diff
@@ -1945,6 +1946,10 @@
   [conn {:keys [::groups ::jobs ::pool] :as ctx}]
   (try
     (log/info "Submitting jobs through raw api:" (map #(dissoc % :command) jobs))
+    (doseq [job jobs]
+      (passport/log-passport-event {"job-uuid" (str (:uuid job))
+                                    "user" (:user job)
```

Comment:
```suggestion
    (doseq [{:keys [user uuid]} jobs]
      (passport/log-passport-event {"job-uuid" (str uuid)
                                    "user" user
```

Reply:
I think destructuring as above would work

Reply:
ah very cool I'll do this